backend/src/channel-message/channel-message.service.ts

```diff
@@ -1,22 +1,22 @@
 import { Injectable } from '@nestjs/common'
-import { ChannelMessage, Prisma } from '@prisma/client'
+import { ChannelMessage } from '@prisma/client'
 import { PrismaService } from '../prisma/prisma.service'
-import {
-  ExceptionChannelMessageTryingToUpdateChannelID,
-  ChannelMessageExceptionTryingToUpdateCreationDate,
-  ChannelMessageExceptionTryingToUpdateID,
-  ChannelMessageExceptionTryingToUpdateSenderID
-} from '../user/exceptions/channel-message.exception'
+import { CreateChannelMessageInput } from './dto/create-channel-message.input'
+import { UpdateChannelMessageInput } from './dto/update-channel-message.input'
 
 @Injectable()
 export class ChannelMessageService {
   constructor(private prisma: PrismaService) {}
 
   async create(
-    data: Prisma.ChannelMessageCreateInput
+    createChannelMessageInput: CreateChannelMessageInput
   ): Promise<ChannelMessage> {
     return this.prisma.channelMessage.create({
```

Comment:
replace createChannelMessageInput by data, we keep the same norm
if return this.prisma.channelMessage.create({ data }) doesn't work then there is a problem on your dto, they must be compatible without re-setting every field on your data